### ISSUE TITLE
Binary input vectors format

### ISSUE THREAD

**Issue by navta** [ahmetaa/fast-dnn#4]
Hi,

Do you have a tool to convert kaldi's vector format to your binary input vectors format?

Thanks

**Comment by ahmetaa** [comment]
No, Java converter only accepts text format for now. I will probably make it directly working with binary kaldi format. But I cannot give a timeline.

**Comment by ahmetaa** [comment]
Sorry I misunderstood your question, I though you were mentioning about neural network formats, my bad. Unfortunately  answer is still no and I think it is a good idea provide this. I will have a look.

**Comment by navta** [comment]
Yes, I used the cpp version. It only accept binary input format. There are text and binary pair input samples in your data/ directory, but I'm not aware of any tool used to convert them. It would be great if you could provide the tool, or just make the fast-dnn to accept text input format.

Thanks
